### PR TITLE
Update to 0.2.0

### PR DESCRIPTION
**Pull request by jonas-schievink**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -29,7 +28,3 @@ rustc-demangle = "0.1.16"
 signal-hook = "0.3.4"
 structopt = "0.3.15"
 hidapi = "1.2.5"
-
-[features]
-# this feature does nothing and it's kept for backwards compatibility; it will be removed in the next semver bump
-defmt = []
```
Comment (japaric) [inline Cargo.toml:35]: we should also remove the `defmt` field form the `Opts` field

**Review by japaric — CHANGES_REQUESTED** [review]
requested a change; the rest looks good to me

File: Cargo.toml (hunk 1 above)
Comment (jonas-schievink) [reply Cargo.toml:35]: Ah, right

**Review by japaric — APPROVED** [review]
:+1:

**Pull request merged**